beta-src/src/webDiplomacyTheme.ts

```diff
@@ -55,6 +55,12 @@ const activeButtonStyle = {
   boxShadow: "0 0 2px 2px #000",
   color: "#000",
 };
+
+const focusButtonStyle = {
+  backgroundColor: "#000",
```

Comment:
i see that we have #fff and #000...let's make variables for these...but don't use a color name because the color might change later. maybe we use something like mainColor and secondaryColor. replace all instances of #fff and #000 in this file with these variables